iSeg/MainWindow.cpp

```diff
@@ -3776,6 +3771,76 @@ void MainWindow::ExecuteExporttissueindex()
 	emit EndDataexport(this);
 }
 
+void MainWindow::LoadTissuelist(const QString& loadfilename, bool append, bool no_popup)
+{
+	tissues_size_t remove_tissues_range;
+	TissueInfos::LoadTissuesReadable(loadfilename.toAscii(), m_Handler3D, remove_tissues_range);
+
+	if (append)
+	{
+		int nr = m_TissueTreeWidget->GetCurrentType() - 1;
+		m_TissueTreeWidget->UpdateTreeWidget();
+
+		tissues_size_t curr_tissue_type = m_TissueTreeWidget->GetCurrentType();
+		if (nr != curr_tissue_type - 1)
+		{
+			TissuenrChanged(curr_tissue_type - 1);
+		}
+	}
+	else // replace
+	{
+		if (remove_tissues_range > 0)
+		{
+			const auto can_delete = [this]() {
+				return QMessageBox::question(this, "iSeg",
+						"Some of the previously existing tissues are\nnot "
+						"contained in the loaded tissue list.\n\nDo you want "
+						"to delete them?",
+						QMessageBox::Yes | QMessageBox::Default, QMessageBox::No);
+			};
+
+			if (no_popup || can_delete() == QMessageBox::Yes)
+			{
+				std::set<tissues_size_t> remove_tissues;
+				for (tissues_size_t type = 1; type <= remove_tissues_range; ++type)
+				{
+					remove_tissues.insert(type);
+				}
+				DataSelection data_selection;
+				data_selection.allSlices = true;
+				data_selection.tissues = true;
+				emit BeginDatachange(data_selection, this, false);
+				m_Handler3D->RemoveTissues(remove_tissues);
+				emit EndDatachange(this, iseg::ClearUndo);
+			}
+		}
+
+		tissues_size_t tissue_count = TissueInfos::GetTissueCount();
+		m_Handler3D->CapTissue(tissue_count);
+
+		m_TissueTreeWidget->UpdateTreeWidget();
+		TissuenrChanged(m_TissueTreeWidget->GetCurrentType() - 1);
+	}
+}
+
+void MainWindow::ReloadMedicalImage(const QString& loadfilename)
+{
+	if (!loadfilename.isEmpty())
+	{
+		DataSelection data_selection;
+		data_selection.allSlices = true;
+		data_selection.bmp = true;
+		data_selection.work = true;
+		data_selection.tissues = true;
+		emit BeginDatachange(data_selection, this, false);
+
+		m_Handler3D->ReloadImage(loadfilename.toAscii(), m_Handler3D->StartSlice());
+		ResetBrightnesscontrast();
+
+		emit EndDatachange(this, iseg::ClearUndo);
+	}
+}
+
 void MainWindow::ExecuteLoadtissues()
```

Comment:
Capital T?

Reply:
well, fixed, but still "wrong" in many similar functions/